src/include/connection.h

```diff
@@ -309,8 +309,17 @@ struct __wt_connection_impl {
 	bool		 evict_server_running;/* Eviction server operating */
 
 	WT_THREAD_GROUP  evict_threads;
-	uint32_t	 evict_threads_max;/* Max eviction threads */
-	uint32_t	 evict_threads_min;/* Min eviction threads */
+	uint32_t         evict_threads_max;/* Max eviction threads */
```

Comment:
We initially did, then thought these fields tune evict_threads group, and would go well with thread related fields.